code-of-conduct.md

```diff
@@ -1,3 +1,3 @@
 # Kubernetes Community Code of Conduct
 
-Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
+Kubernetes follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
This is as per https://github.com/kubernetes/kubernetes-template-project/blob/master/code-of-conduct.md.

Closing.
/close